scripts/comparison/create_report.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -xe
+
+HTML_REP_SCRIPT="$(pwd)/scripts/comparison/html_parser.py"
```

Comment:
never use pwd for paths relative to the file you are executing.
use: BASEDIR=$(dirname $0) instead